src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Media/Generated/DoubleCollection.cs

```diff
@@ -723,7 +723,7 @@ internal string ConvertToString(string format, IFormatProvider provider)
 
                 if (i != _collection.Count-1)
                 {
-                    str.Append(" ");
```

Comment:
You should revert the changes made in generated files (See [this list of generated files](https://github.com/dotnet/wpf/blob/main/eng/WpfArcadeSdk/tools/WPF_Generated_Files.txt)). These files are generated by MilCodeGen which is not in a state to be modified or runned in this repo. #6135 aims to allow to modify and run MilCodeGen which would allow to apply this change [here](https://github.com/dotnet/wpf/blob/main/src/Microsoft.DotNet.Wpf/src/WpfGfx/codegen/mcg/generators/ManagedStruct.cs#L375) and add proper `<auto-generated>` file header.

Reply:
The fact that these are generated doesn't seem to be a good reason not to make such changes.  I've updated the templates accordingly.  When they again become runnable, my changes shouldn't result in diffs.